src/util/parseoptions.cpp

```diff
@@ -17,13 +17,34 @@ Author: Daniel Kroening, kroening@kroening.com
 #include <util/cmdline.h>
 #include <util/parseoptions.h>
 #include <util/signal_catcher.h>
+#include <boost/program_options.hpp>
 
+using namespace boost::program_options;
+#include <ac_config.h>
```

Comment:
Why do you need this header?

Reply:
To show ESBMC_VERSION  before help message.

Reply:
But you don't need it anymore, right? Nothing is using ESBMC_VERSION in this file.